.gitignore

```diff
@@ -9,3 +9,4 @@ paper.run.xml
 *out
 *.ipynb_checkpoints*
 .DS_Store
+tex/_minted-paper
```

Comment:
I think we should also add the following:

paper.pdf
paper.pyg
paper.synctex.gz